src/vs/workbench/contrib/files/browser/fileCommands.ts

```diff
@@ -236,11 +236,10 @@ async function resourcesToClipboard(resources: URI[], relative: boolean, clipboa
 		const lineDelimiter = isWindows ? '\r\n' : '\n';
 
 		let separator: '/' | '\\' | undefined = undefined;
-		if (relative) {
-			const relativeSeparator = configurationService.getValue('explorer.copyRelativePathSeparator');
```

Comment:
We cannot just delete settings like this. If we took in this feature it would be an additional setting that works alongside the relative path setting